ts/Series/ArcDiagram/ArcDiagramSeries.ts

```diff
@@ -28,6 +28,9 @@ import Series from '../../Core/Series/Series.js';
 import SeriesRegistry from '../../Core/Series/SeriesRegistry.js';
 import SVGRenderer from '../../Core/Renderer/SVG/SVGRenderer.js';
 import U from '../../Core/Utilities.js';
+import Highcharts from '../../Core/Globals.js';
+import TextPath from '../../Extensions/TextPath.js';
+TextPath.compose((Highcharts as AnyRecord).SVGElement);
```

Comment:
Alright, done!

Those series which used `import type SVGElement` were necessary to change to just `import SVGElement` since we cannot import `SVGElement` twice